SEP-0000.md

```diff
@@ -0,0 +1,29 @@
+# SEP-0000 -- Index of SunPy Enhancement Proposals (SEPs)
+
+| SEP           | 0 |
```

Comment:
Yep, just following what the peps were doing. I could add a header row to fix this problem,